examples/SJ_GaAs_external_absorber.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy as sp
+
+from solcore.solar_cell import SolarCell
+from solcore.structure import Layer, Junction, TunnelJunction
+from solcore import material
+from solcore.light_source import LightSource
+from solcore.solar_cell_solver import solar_cell_solver
+
+## We can use the derivative in this case but we are going 
+## use the absorption, as typically optical softwares will give it.
+def single_pass(material, wl,  d, I0=1.):
+    """It returns the absorption using the Beer-Lambert law,
+    to change the number of passes d=d0*N
+    Args:
+        material (solcore.material):  Complex refractive index (n + 1j * K)
+        wl (np.array): Wavelength (wl)
+        d (np.float): Material thickness (microns)
+        I0 (np.float/np.array): Intensity at the interface.
+    Returns:
+        np.array
+    """
+    K = material.k(wl)
+    alpha = 4 * np.pi * K/ wl
+    return I0 * (1. - np.exp(-alpha * d))
+
+def convert_cell_to_layers(solar_cell):
+    """ It converts a solar_cell object to a list of layers.
+    Args:
+        solar_cell (solcore.SolarCell)
+    Return:
+        [(solcore.Layer), ...]
+    """
+    all_layers = []
+    for j, layer_object in enumerate(solar_cell):
+        if type(layer_object) is Layer:
+            all_layers.append(layer_object)
+        elif type(layer_object) in [TunnelJunction, Junction]:
+            for i, layer in enumerate(layer_object):
+                all_layers.append(layer)
+    return all_layers
+
+
+
+## Define a simple GaAs cell
+
+nm = 1e-9
+T = 298
+NP = 128
+meshpoints = -400
```

Comment:
Is not a big deal, but I don't think you use the mesh options in the example. for the sake of cleanliness, it might be best to remove it.